corehq/apps/integration/kyc/models.py

```diff
@@ -211,16 +211,16 @@ def kyc_verification_status(self):
     def kyc_provider(self):
         return self.user_data.get('kyc_provider')
 
-    def update_verification_status(self, is_verified, device_id=None):
+    def update_verification_status(self, verification_status, device_id=None):
         from corehq.apps.hqcase.utils import update_case
 
-        assert is_verified in [True, False]
+        assert verification_status in [True, False]
```

Comment:
nit: would be nice to read `KycVerificationStatus.PASSED, KycVerificationStatus.FAILED` here instead of `True, False`

Reply:
Done in the following commit.